visual_behavior/data_access/loading.py

```diff
@@ -451,6 +451,8 @@ def get_extened_stimulus_presentations(stimulus_presentations, licks, rewards, r
                                           stimulus_presentations.rewards.values]
     stimulus_presentations['reward_rate'] = stimulus_presentations['rewarded'].rolling(window=320, min_periods=1,
                                                                                        win_type='triang').mean()
+    # not sure why the output of this is two orders of magnitude off from expected units of rewards/min...
+    stimulus_presentations['reward_rate'] = stimulus_presentations['reward_rate'] * 100
```

Comment:
Because line 452 computes reward rate in terms of rewards/image-presentation.

To convert to rewards/minute we should multiple by (60s/.75s) = 80

Reply:
Before we change the units, let's make sure that functions downstream of this expect reward rate in the proper units. I know I often work in rewards/image-presentation.

Reply:
Maybe we should just include both `reward_rate_per_minute`, and `reward_rate_per_image`

Reply:
I just noticed that `lick_rate` has a `/.75` which I think puts lick rate into licks/sec. We should be very careful about units here. I know the engagement state (where is it computed??) definition that doug and I developed uses the lick-bout-rate (computed where??) and reward-rate variables

Reply:
thanks for the correction. 

i am pretty sure that all of my downstream analyses expect things in rewards per minute. let me know if you have anything downstream that expects rewards/image presentation.

 perhaps we should include it both ways, but then one or the other of us will have to deal with renaming that column in our analysis.

Reply:
i believe these concerns about reward_rate calculation are addressed in this PR: https://github.com/AllenInstitute/visual_behavior_analysis/pull/798

Reply:
Kinda confusing that you are fixing one PR with another, but #798 does resolve this issue